### PR TITLE
HEEDLS-305 Post learning assessment next button back end

### PR DESCRIPTION
**Pull request by kwunkiuma**
Add query and tests.

Ran and passed all unit tests.

### CODE REVIEW

File: DigitalLearningSolutions.Data.Tests/Services/PostLearningAssessmentTests.cs [hunk 1]
```diff
@@ -20,7 +23,9 @@ public void Setup()
             var logger = A.Fake<ILogger<PostLearningAssessmentService>>();
             postLearningAssessmentService = new PostLearningAssessmentService(connection, logger);
             postLearningAssessmentTestHelper = new PostLearningAssessmentTestHelper(connection);
-        }
+            sectionContentTestHelper = new SectionContentTestHelper(connection);
+            tutorialContentTestHelper = new TutorialContentTestHelper(connection);
+    }
```
Comment (FridaTveit) [inline DigitalLearningSolutions.Data.Tests/Services/PostLearningAssessmentTests.cs:28]: Indentation off

File: DigitalLearningSolutions.Data.Tests/Services/PostLearningAssessmentTests.cs [hunk 2]
```diff
@@ -171,6 +178,249 @@ int progressId
             result.PostLearningLocked.Should().Be(scoresReturnedFromOldStoredProcedure.PLLocked);
         }
 
+        [Test]
+        public void Get_post_learning_assessment_next_section_id_should_return_id_if_at_start_of_list()
```
Comment (FridaTveit) [inline DigitalLearningSolutions.Data.Tests/Services/PostLearningAssessmentTests.cs:182]: What's at the start of what list? 🙂

**Review by FridaTveit — COMMENTED** [review]
A few comments, otherwise looks good. Probably worth @benjimarshall and @SChandarana having a quick look too as they've been working on the same query for the section page 🙂

File: DigitalLearningSolutions.Data.Tests/Services/PostLearningAssessmentTests.cs (hunk 2 above)
Comment (kwunkiuma) [reply DigitalLearningSolutions.Data.Tests/Services/PostLearningAssessmentTests.cs:182]: This test is referring to the first section in a customisation. This is the same for the other tests that mention list in the name. Would you like a more descriptive name?

Comment (FridaTveit) [reply DigitalLearningSolutions.Data.Tests/Services/PostLearningAssessmentTests.cs:182]: Yes please, this was a comment which applies to all the tests which mention list in the name 🙂 Maybe something like `Get_post_learning_assessment_for_first_section_in_course_has_next_section`? 🙂

**Comment by benjimarshall** [comment]
This looks good to me, and doesn't conflict with any of the changes I've made I don't think 👍

**Review by FridaTveit — APPROVED** [review]
Looks good 🙂

**Pull request merged**